packages/moda-sdk/src/profile/createProfile.ts

```diff
@@ -0,0 +1,33 @@
+import { Hash } from 'viem';
+import { ProfileABI } from './abi/ProfileABI';
+import { profileAddress } from './constants/profileAddress';
+import { CreateProfileArgs } from './types';
+
+export async function createProfile(
+  createProfileArgs: CreateProfileArgs,
+): Promise<Hash> {
+  const { config, metadataUri } = createProfileArgs;
+  const { publicClient, walletClient } = config;
+
+  if (publicClient === undefined) {
+    throw new Error('Public client is undefined');
+  }
+
+  if (walletClient === undefined) {
+    throw new Error('Wallet client is undefined');
+  }
+
+  const [account] = await walletClient.getAddresses();
+
+  const { request } = await publicClient.simulateContract({
+    account,
+    abi: ProfileABI,
+    address: profileAddress,
```

Comment:
Let's inject this in the Config so that we can use it for different chains. e.g. Mainnet, Polygon, etc